docker-compose.yml

```diff
@@ -1,238 +1,220 @@
-      DCR_SYNC_DATASET_SUBJECT: 'http://data.lblod.info/datasets/delta-producer/dumps/lblod-harvester/BesluitenCacheGraphDump'
-      DCR_INITIAL_SYNC_JOB_OPERATION: 'http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/initialSync/besluiten-3'
-      DCR_DELTA_SYNC_JOB_OPERATION: 'http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/besluitenSyncing-3'
-      DCR_JOB_CREATOR_URI: 'http://data.lblod.info/services/id/besluiten-consumer-3'
-      DCR_DISABLE_INITIAL_SYNC: false
-      DIRECT_DATABASE_ENDPOINT: 'database'
+      DCR_SERVICE_NAME: consumer-3
+      DCR_CRON_PATTERN_DELTA_SYNC: 0 * * * * # Every hour
+      DCR_START_FROM_DELTA_TIMESTAMP: 2024-10-01
+      DCR_SYNC_BASE_URL: https://lokaalbeslist-harvester-3.s.redhost.be/
+      DCR_SYNC_FILES_PATH: /sync/besluiten/files
+      DCR_SYNC_DATASET_SUBJECT: http://data.lblod.info/datasets/delta-producer/dumps/lblod-harvester/BesluitenCacheGraphDump
+      DCR_INITIAL_SYNC_JOB_OPERATION: http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/initialSync/besluiten
+      DCR_DELTA_SYNC_JOB_OPERATION: http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/besluitenSyncing
+      DCR_JOB_CREATOR_URI: http://data.lblod.info/services/id/besluiten-consumer
+      DCR_DISABLE_INITIAL_SYNC: true
+      DCR_WAIT_FOR_INITIAL_SYNC: false
+      DCR_DISABLE_DELTA_INGEST: false
+      INGEST_GRAPH: http://mu.semte.ch/graphs/public
       BATCH_SIZE: 500
   ######################
-  # Producer services ##
+  # LDES              ##
   ######################
-  jobs-dashboard:
-    # Note: the following dashboard is a bit of a stretch to use here; since intially tailored towards harvesting
-    #  but it uses the common job model and has a sparql interface; so it covers a lot of useful features.
-    #  Somday; I geuss we'll have a cleaner version
-    image: lblod/frontend-harvesting-self-service:2.1.5
-    profiles:
-      - frontend
-    volumes:
-      - ./config/frontend:/config
-    labels:
-      - 'logging=true'
-    restart: always
-    logging: *default-logging
-    environment:
-      EMBER_AUTHENTICATION_ENABLED: false
-      EMBER_BESLUITEN_HARVESTING_ENABLED: false
-      EMBER_WORSHIP_HARVESTING_ENABLED: false
-  publication-triplestore:
-    image: redpencil/virtuoso:1.2.0-rc.1
-    environment:
-      SPARQL_UPDATE: true
-      DEFAULT_GRAPH: 'http://mu.semte.ch/application'
-    volumes:
-      - ./data/publication-triplestore:/data
-      - ./config/publication-triplestore/virtuoso-qa.ini:/data/virtuoso.ini
-      - ./config/publication-triplestore/:/opt/virtuoso-scripts
-    labels:
-      - 'logging=true'
-    restart: always
-  jobs-controller:
-    image: lblod/job-controller-service:1.0.1
-    volumes:
-      - ./config/jobs-controller/:/config/
-    labels:
-      - 'logging=true'
+  ldes-backend:
+    image: redpencil/fragmentation-producer:0.4.1
     restart: always
     logging: *default-logging
```

Comment:
this should be set for every service, also for the consumers
same for labels:
- logging=true

Reply:
Updated https://github.com/lblod/app-mandatees-decisions/pull/11/commits/22787585fea37ab218ec3991001793cbdea4f68a